data/conversation/workspaces/banking.json

```diff
@@ -1 +1 @@
:{"text":{"values":["Please click <a href=\"http://www.ideacellular.com/\" target=\"_blank\">here</a> for the account opening process"],"selection_policy":"sequential"}},"parent":"node_11_1488861036277","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@account_type:savings","description":null,"dialog_node":"node_12_1488861096661","previous_sibling":"node_2_1492588643560"},{"go_to":null,"title":null,"output":{"text":{"values":["Please click on the \"Yes\" button above."],"selection_policy":"sequential"}},"parent":"node_1_1479300011534","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@acknowledgement:yes || $res=='YES'","description":null,"dialog_node":"node_2_1479300155087","previous_sibling":null},{"go_to":null,"title":null,"output":{"text":{"values":["Please click \"Yes\" button above."],"selection_policy":"sequential"}},"parent":"node_2_1492588643560","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@acknowledgement:yes","description":null,"dialog_node":"node_4_1492588676901","previous_sibling":null},{"go_to":null,"title":null,"output":{"text":{"values":["Please confirm with a yes or no confirmation."],"selection_policy":"sequential"}},"parent":"home loan","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"true","description":null,"dialog_node":"node_4_1484321533426","previous_sibling":"node_7_1477061727761"},{"go_to":null,"title":null,"output":{"text":{"values":["Please contact merchant Customer Care for refund/delivery/order status"],"selection_policy":"sequential"}},"parent":"node_8_1488858811196","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@features:status","description":null,"dialog_node":"node_25_1486267658814","previous_sibling":"node_26_1486269385463"},{"go_to":null,"title":null,"output":{"text":{"values":["Please contact the Merchant Customer Care for queries related to payment and product."],"selection_policy":"sequential"}},"parent":"node_5_1486279608967","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@features:product || @option:payment || @account_type:merchant","description":null,"dialog_node":"node_10_1486280199008","previous_sibling":"node_7_1486279676925"},{"go_to":null,"title":null,"output":{"text":{"values":["Please enter a valid PAN number."],"selection_policy":"sequential"}},"parent":"node_1_1492426752086","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"true","description":null,"dialog_node":"node_11_1492429161945","previous_sibling":"node_3_1492427252795"},{"go_to":null,"title":null,"output":{"text":{"values":["Please enter the amount.
```

Comment:
the beginning of this file has some garbage we can probably trim out

```
{"intent":"nonsense","examples":[{"text":"ajshajhds"},{"text":"is the sky blue"},{"text":"hashahah"},{"text":"hahahaha"},{"text":"hahahah"},{"text":"come here"},{"text":"ccc"},{"text":"buffalo"},{"text":"are you mad"},{"text":"are you a fool"},{"text":"this is red"},{"text":"sdjhsjd"},{"text":"savings"},{"text":"it is bad"}],"description":null}
```

Reply:
++ the conditional referencing `#nonsense` would need to be revised as well.